lobby-client-http/src/main/java/org/triplea/http/client/github/issues/GithubClient.java

```diff
@@ -0,0 +1,44 @@
+package org.triplea.http.client.github.issues;
+
+import java.net.URI;
+import java.util.Map;
+
+import org.triplea.http.client.HttpClient;
+import org.triplea.http.client.error.report.ErrorUploadRequest;
+import org.triplea.http.client.github.issues.create.CreateIssueResponse;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import feign.FeignException;
+import feign.HeaderMap;
+import feign.Headers;
+import feign.Param;
+import feign.RequestLine;
+
+@SuppressWarnings("InterfaceNeverImplemented")
```

Comment:
As already mentioned in a different PR; It would be nice if a comment would point out where exactly a implementation of this instance is created.

Reply:
Concluding from the other convo, I disagree as a reader just needs to continue looking a few lines lower to find the answer to that. A search for the interface name or usages would find the answer. Construction from there is not standard, but it is traceable to the a feign builder. From there one does need to know that feign builder constructs an http client, there is some level of knowledge that does need to be known. The point of focus for documentation should be when a reader gets to the feign builder so they can understand that and have links to feign documentation in the one location.